### PR TITLE
First commit User Screen

### PR DESCRIPTION
**Pull request by soundrob**

### CODE REVIEW

File: app/src/main/java/com/endava/myendava/activities/UsersActivity.java [hunk 1]
```diff
@@ -55,23 +61,45 @@ protected void onCreate(Bundle savedInstanceState) {
...
+                icon = getResources().getDrawable(R.drawable.icons_filled_tags);
+                break;
+            case CATEGORY_LOGISTICS:
+                icon = getResources().getDrawable(R.drawable.ic_logistics);
+                break;
+            default:
+                break;
+        }
+        tagTypeTextView.setCompoundDrawablesWithIntrinsicBounds(icon, null, null, null);
+        if (tag.getSubcategory().equals(CATEGORY_SOFT)) {
```
Comment (gabrielblaj07) [inline app/src/main/java/com/endava/myendava/activities/UsersActivity.java]: I think you copied a little too much code from the tags adapter ^^

File: app/src/main/java/com/endava/myendava/adapters/UsersAdapter.java [hunk 2]
```diff
@@ -54,17 +56,16 @@ public void setData(List<User> users) {
 
     class UserViewHolder extends RecyclerView.ViewHolder {
 
-        View itemView;
+        @BindView(R.id.user_photo_image_view)
         CircleImageView photoImageView;
```
Comment (gabrielblaj07) [inline app/src/main/java/com/endava/myendava/adapters/UsersAdapter.java]: no need to specify the component on all namings.. here is pretty clear that the photo is an ImageView while the name and grade is a TextView

File: app/src/main/java/com/endava/myendava/activities/UsersActivity.java [hunk 3]
```diff
@@ -38,12 +42,14 @@
 
     @BindView(R.id.tag_description_text_view)
     TextView descriptionTextView;
-
-    @BindView(R.id.collapsing_toolbar_layout)
-    CollapsingToolbarLayout collapsingToolbarLayout;
-
+    @BindView(R.id.tag_title)
+    TextView tagTitleTextView;
```
Comment (gabrielblaj07) [inline app/src/main/java/com/endava/myendava/activities/UsersActivity.java]: no need to specify the component name..

File: app/src/main/java/com/endava/myendava/activities/UsersActivity.java [hunk 4]
```diff
@@ -55,23 +61,45 @@ protected void onCreate(Bundle savedInstanceState) {
...
-        ActionBar actionBar = getSupportActionBar();
-        if (actionBar != null) {
-            actionBar.setTitle(tag.getTagName());
-            actionBar.setDisplayHomeAsUpEnabled(true);
+        tagTitleTextView.setText(tag.getTagName());
+        backPressImageViewButton.setOnClickListener(view -> onBackPressed());
+        descriptionTextView.setText(tag.getTagDescription());
+        tagTypeTextView.setText(tag.getCategory());
+        Drawable icon = null;
+        switch (tag.getSubcategory()) {
```
Comment (gabrielblaj07) [inline app/src/main/java/com/endava/myendava/activities/UsersActivity.java:70]: you can move this in a new method

File: app/src/main/res/layout/activity_users.xml [hunk 5]
```diff
@@ -1,69 +1,103 @@
...
+        app:layout_behavior="com.google.android.material.appbar.AppBarLayout$ScrollingViewBehavior"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent">
 
         <androidx.constraintlayout.widget.ConstraintLayout
             android:id="@+id/container"
             android:layout_width="match_parent"
             android:layout_height="match_parent">
 
+            <ImageView
```
Comment (gabrielblaj07) [inline app/src/main/res/layout/activity_users.xml:37]: too much space between the status bar and this icon

File: app/src/main/java/com/endava/myendava/adapters/UsersAdapter.java (hunk 2 above)
Comment (soundrob) [reply app/src/main/java/com/endava/myendava/adapters/UsersAdapter.java]: i,ve changed it to userImage, userName and userGrade

File: app/src/main/java/com/endava/myendava/activities/UsersActivity.java [hunk 6]
```diff
@@ -37,13 +41,15 @@
...
-
+    TextView tagDescription;
+    @BindView(R.id.tag_title)
+    TextView tagTitle;
+    @BindView(R.id.tag_type)
+    TextView tagType;
     @BindView(R.id.progress_bar)
     ProgressBar mProgressBar;
+    @BindView(R.id.arrow_back_button)
+    ImageView backPressImageViewButton;
```
Comment (grauralex92) [inline app/src/main/java/com/endava/myendava/activities/UsersActivity.java]: backImageView

File: app/src/main/java/com/endava/myendava/adapters/UsersAdapter.java [hunk 7]
```diff
@@ -54,23 +56,22 @@ public void setData(List<User> users) {
 
     class UserViewHolder extends RecyclerView.ViewHolder {
 
-        View itemView;
-        CircleImageView photoImageView;
-        TextView nameTextView;
-        TextView gradeTextView;
+        @BindView(R.id.user_photo_image_view)
+        CircleImageView userImage;
```
Comment (grauralex92) [inline app/src/main/java/com/endava/myendava/adapters/UsersAdapter.java]: userImageTextView

File: app/src/main/java/com/endava/myendava/adapters/UsersAdapter.java [hunk 8]
```diff
@@ -54,23 +56,22 @@ public void setData(List<User> users) {
...
+        @BindView(R.id.user_name_text_view)
+        TextView userName;
```
Comment (grauralex92) [inline app/src/main/java/com/endava/myendava/adapters/UsersAdapter.java]: userNameTextView

File: app/src/main/java/com/endava/myendava/adapters/UsersAdapter.java [hunk 9]
```diff
@@ -54,23 +56,22 @@ public void setData(List<User> users) {
...
+        @BindView(R.id.user_grade_text_view)
+        TextView userGrade;
```
Comment (grauralex92) [inline app/src/main/java/com/endava/myendava/adapters/UsersAdapter.java]: userGradeTextView

**Review by grauralex92 — CHANGES_REQUESTED** [review]

**Review by grauralex92 — APPROVED** [review]

**Pull request merged**